src/TorchSharp/NN/ReLu.cs

```diff
@@ -16,12 +16,9 @@ internal ReLU(bool inPlace = false) : base()
             _inPlace = inPlace;
         }
 
-        [DllImport("LibTorchSharp")]
-        private static extern IntPtr THSNN_reluApply(IntPtr tensor, bool inPlace);
```

Comment:
This signature doesn't match the signature in the native DLL (which has no inPlace argument)